### PR TITLE
Translation from english to spanish of file pipeline_tutorial.mdx

### PR DESCRIPTION
**Pull request by FernandoLpz**
## Translation from English to Spanish of file pipeline_tutorial.mdx

Fixes #15947 
@omarespejel

I added the translation of the `pipeline_tutorial.mdx` file

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.

### CODE REVIEW

**Comment by omarespejel** [comment]
Thanks @FernandoLpz! Will review it :)

File: docs/source_es/pipeline_tutorial.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+```py
+>>> generator("Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone")
+[{'generated_text': 'Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone, Seven for the Dragon-lords (for them to rule in a world ruled by their rulers, and all who live within the realm'}]
+```
+
+## Pipeline de audio
+
+La flexibilidad del [`pipeline`] significa que también se puede extender a tareas de audio.
+
+Por ejemplo, clasifiquemos la emoción de un breve fragmento del famoso discurso de John F. Kennedy ["We choose to go to the Moon"](https://en.wikipedia.org/wiki/We_choose_to_go_to_the_Moon). Encuentra un modelo de [audio classification](https://huggingface.co/models?pipeline_tag=audio-classification) en Model Hub para el reconocimiento de emociones y cárgualo en el [`pipeline`]:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:96]:
```suggestion
Por ejemplo, clasifiquemos la emoción de un breve fragmento del famoso discurso de John F. Kennedy ["We choose to go to the Moon"](https://en.wikipedia.org/wiki/We_choose_to_go_to_the_Moon). Encuentra un modelo de [audio classification](https://huggingface.co/models?pipeline_tag=audio-classification) en Model Hub para el reconocimiento de emociones y cárgalo en el [`pipeline`]:
```

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thank you! Some nits

**Comment by FernandoLpz** [comment]
Your welcome!

I just fix the typos. 

@omarespejel

File: docs/source_es/pipeline_tutorial.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+Un [pipeline] simplifica el uso de cualquier modelo de [Model Hub](https://huggingface.co/models) para la inferencia en una variedad de tareas, como la generación de texto, la segmentación de imágenes y la clasificación de audio. Incluso si no tienes experiencia con una modalidad específica o no comprendes el código que alimenta los modelos, ¡aún puedes usarlos con el [pipeline]! Este tutorial te enseñará a:
+
+* Utilizar un [`pipeline`] para inferencia.
+* Utilizar un tokenizador o modelo específico.
+* Utilizar un [`pipeline`] para tareas de audio y visión.
+
+<Tip>
+
+Echa un vistazo a la documentación del [`pipeline`] para obtener una lista completa de tareas admitidas.
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:23]:
```suggestion
Echa un vistazo a la documentación de [`pipeline`] para obtener una lista completa de tareas admitidas.
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Comienza creando un [`pipeline`] y específica una tarea de inferencia:
+
+```py
+>>> from transformers import pipeline
+
+>>> generator = pipeline(task="text-generation")
+```
+
+2. Pasa tu texto de entrada a el [`pipeline`]:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:39]:
```suggestion
2. Pasa tu texto de entrada al [`pipeline`]:
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+```py
+>>> generator(
+...     [
+...         "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone",
+...         "Nine for Mortal Men, doomed to die, One for the Dark Lord on his dark throne",
+...     ]
+... )
+```
+
+Cualquier parámetro adicional para tu tarea también se puede incluir en el [`pipeline`]. La tarea `text-generation` tiene un método [`~generation_utils.GenerationMixin.generate`] con varios parámetros para controlar la salida. Por ejemplo, si deseas generar más de una salida, definelo en el parámetro `num_return_sequences`:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:57]:
```suggestion
Cualquier parámetro adicional para tu tarea también se puede incluir en el [`pipeline`]. La tarea `text-generation` tiene un método [`~generation_utils.GenerationMixin.generate`] con varios parámetros para controlar la salida. Por ejemplo, si deseas generar más de una salida, defínelo en el parámetro `num_return_sequences`:
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+```py
+>>> generator(
+...     "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone",
+...     num_return_sequences=2,
+... )
+```
+
+### Selecciona un modelo y un tokenizador
+
+El [`pipeline`] acepta cualquier modelo de [Model Hub](https://huggingface.co/models). Hay etiquetas en Model Hub que te permiten filtrar por modelo que te gustaría utilizar para tu tarea. Una vez que hayas elegido un modelo apropiado, cárgalo con la clase `AutoModelFor` y [`AutoTokenizer'] correspondientes. Por ejemplo, carga la clase [`AutoModelForCausalLM`] para una tarea de modelado de lenguaje causal:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:68]:
```suggestion
El [`pipeline`] acepta cualquier modelo del [Model Hub](https://huggingface.co/models). Hay etiquetas en el Model Hub que te permiten filtrar por el modelo que te gustaría utilizar para tu tarea. Una vez que hayas elegido un modelo apropiado, cárgalo con la clase `AutoModelFor` y [`AutoTokenizer'] correspondientes. Por ejemplo, carga la clase [`AutoModelForCausalLM`] para una tarea de modelado de lenguaje causal:
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+```py
+>>> from transformers import AutoTokenizer, AutoModelForCausalLM
+
+>>> tokenizer = AutoTokenizer.from_pretrained("distilgpt2")
+>>> model = AutoModelForCausalLM.from_pretrained("distilgpt2")
+```
+
+Crea un [`pipeline`] para tu tarea y específica el modelo y el tokenizador que has cargado:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:77]:
```suggestion
Crea un [`pipeline`] para tu tarea y específica el modelo y el tokenizador que cargaste:
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+Pasa tu texto de entrada a [`pipeline`] para generar algo de texto:
+
+```py
+>>> generator("Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone")
+[{'generated_text': 'Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone, Seven for the Dragon-lords (for them to rule in a world ruled by their rulers, and all who live within the realm'}]
+```
+
+## Pipeline de audio
+
+La flexibilidad del [`pipeline`] significa que también se puede extender a tareas de audio.
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:94]:
```suggestion
La flexibilidad de [`pipeline`] significa que también se puede extender a tareas de audio.
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+ {'label': 'happy', 'score': 0.12635163962841034},
+ {'label': 'angry', 'score': 0.12439591437578201},
+ {'label': 'fearful', 'score': 0.12404385954141617}]
+```
+
+## Pipeline de visión
+
+Finalmente, utilizar un [`pipeline`] para tareas de visión es prácticamente idéntico.
+
+Especifica tu tarea de visión y pasa tu imagen al clasificador. La imagen puede ser un enlace o una ruta local a la imagen. Por ejemplo, ¿qué especie de gato se muestra a continuación?
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:121]:
```suggestion
Específica tu tarea de visión y pasa tu imagen al clasificador. La imagen puede ser un enlace o una ruta local a la imagen. Por ejemplo, ¿qué especie de gato se muestra a continuación?
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+```py
+>>> from transformers import pipeline
+
+>>> audio_classifier = pipeline(
+...     task="audio-classification", model="ehcalabres/wav2vec2-lg-xlsr-en-speech-emotion-recognition"
+... )
+```
+
+Pasa el archivo de audio a el [`pipeline`]:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:106]:
```suggestion
Pasa el archivo de audio al [`pipeline`]:
```

File: docs/source_es/pipeline_tutorial.mdx [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+```py
+>>> generator("Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone")
+[{'generated_text': 'Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone, Seven for the Dragon-lords (for them to rule in a world ruled by their rulers, and all who live within the realm'}]
+```
+
+## Pipeline de audio
+
+La flexibilidad del [`pipeline`] significa que también se puede extender a tareas de audio.
+
+Por ejemplo, clasifiquemos la emoción de un breve fragmento del famoso discurso de John F. Kennedy ["We choose to go to the Moon"](https://en.wikipedia.org/wiki/We_choose_to_go_to_the_Moon). Encuentra un modelo de [audio classification](https://huggingface.co/models?pipeline_tag=audio-classification) en Model Hub para el reconocimiento de emociones y cárgalo en el [`pipeline`]:
```
Comment (omarespejel) [inline docs/source_es/pipeline_tutorial.mdx:96]:
```suggestion
Por ejemplo, clasifiquemos la emoción de un breve fragmento del famoso discurso de John F. Kennedy ["We choose to go to the Moon"](https://en.wikipedia.org/wiki/We_choose_to_go_to_the_Moon). Encuentra un modelo de [audio classification](https://huggingface.co/models?pipeline_tag=audio-classification) para reconocimiento de emociones en el Model Hub y cárgalo en el [`pipeline`]:
```

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thanks! Just some last nits.

**Comment by omarespejel** [comment]
Hi @sgugger! It seems to me that the failed checks are due to something else and not this doc. WDYT?

**Review by sgugger — APPROVED** [review]
All tests are failing because of the rename of the branch from master to main, but the code is not adapted. A rebase would fix them all, but it's not necessary since we know this PR only touches doc files.

Thanks a lot for working on this!

**Pull request merged**